src/wamr/wamr.cc

```diff
@@ -354,7 +356,7 @@ bool Wamr::link(std::string_view /*debug_name*/) {
     case WASM_EXTERN_FUNC: {
       WasmFuncPtr func = wasm_func_copy(wasm_extern_as_func(actual_extern));
       const wasm_name_t *name_ptr = wasm_exporttype_name(export_types.get()->data[i]);
-      module_functions_.insert_or_assign(std::string(name_ptr->data, name_ptr->size),
+      module_functions_.insert_or_assign(std::string(name_ptr->data, std::strlen(name_ptr->data)),
```

Comment:
Same here, I think you should remove the second parameter in `std::string()`'s constructor.